dr-solution-introduction.md

```diff
@@ -0,0 +1,113 @@
+
+This architecture is simple and highly available with region-level error tolerance objective, scalable write capability, second-level RPO, and minute-level RTO or even lower. If a production system does not require the RPO to be zero, this DR solution is recommended. For more information about this solution, see [DR solution based on primary and secondary clusters](/dr-secondary-cluster.md).
+
+### DR solution based on multiple replicas in a single cluster
+
+![Multi-replica cluster DR](/media/dr/multi-replica-dr.png)
+
+In the preceding architecture, each region has two complete data replicas located in different available zones (AZs). The entire cluster is across three regions. Region 1 is the primary region that handles read and write requests. When region 1 is completely unavailable due to a disaster, region 2 can be used as a DR region. Region 3 is a replica used to meet the majority protocol. This architecture is also called the "2-2-1" solution.
+
+This solution provides region-level error tolerance, scalable write capability, zero RPO, and minute-level RTO or even lower. If a production system requires zero RPO, it is recommended to use this DR solution. For more information about this solution, see [DR solution based on multiple replicas in a single cluster](/dr-multi-replica.md).
+
+### DR solution based on TiCDC and multiple replicas
+
+The preceding two solutions provide regional DR. However, they fail to work if multiple regions are unavailable at the same time. If your system is very important and requires error tolerance objective to cover multiple regions, you need to combine these two solutions.
+
+![TiCDC-based multi-replica cluster DR](/media/dr/ticdc-multi-replica-dr.png)
+
+In the preceding architecture, there are two TiDB clusters. Cluster 1 has five replicas that span three regions. Region 1 contains two replicas that work as the primary region and handle write requests. Region 2 has two replicas that work as the DR region for region 1. This region provides read services that are not sensitive to latency. Located in Region 3, the last replica is used for voting.
+
+As the DR cluster for region 1 and region 2, cluster 2 runs in region 3 and contains three replicas. TiCDC replicates data from cluster 1. This architecture looks complicated but it can increase the error tolerance objective to multiple regions. If the RPO is not required to be zero when multiple regions are unavailable at the same time, this architecture is a good choice. This architecture is also called the "2-2-1:1" solution.
+
+Of course, if the error tolerance objective is multiple regions and RPO must be zero, you can also consider creating a cluster with at least nine replicas spanning five regions. This architecture is also called the "2-2-2-2-1" solution.
+
+### DR solution based on BR
```

Comment:
What are the version requirements for this? Do both Cluster1 and Cluster2 have to be the exact same version? Or only the same major version? How would upgrades work?

Reply:
@allengaoo PTAL

Reply:
I assume  this comment is related to TiCDC based DR solution. Currently, we highly recommend customer use version 6.5.1 for tidb and ticdc.  For cluster 1 and 2 it is highly recommend they have same major version.  

I will consider to add upgrade related information later.